### ISSUE TITLE
Windows Podman Machine Restart Failed with podman 4.8.0

### ISSUE THREAD

**Issue by odockal** [containers/podman-desktop#5129]
### Bug description

I have a custom podman-wsl-fedora image with podman 4.8.0.
Restart of the podman machine fails.
 
![podman_machine_restart_failed_windows](https://github.com/containers/podman-desktop/assets/19164299/6078e95f-9232-4be9-a0a5-d550e9848fc5)

### Operating system

Windows 10

### Installation Method

Installer from website/GitHub releases

### Version

next (development version)

### Steps to reproduce

It seems that problematic is restarting the machine from the Machine's details page.
1. initialize and start podman machine
2. Assert: Machine is up and running
3. Go to the details page of the machine
4. Restart Machine
5. Assert: No error, machine is stopped and started again
Actual result: Error appears, although in the machine log it seems that machine went to STOP state and then eventually into STARTED. So it seems like UI issue more than actual problem with machine restart operation.

### Relevant log output

```shell
Podman Machine Log:
Machine "podman-machine-default" stopped successfully

Starting machine "podman-machine-default"

your 131072x1 screen size is bogus. expect trouble


This machine is currently configured in rootless mode. If your containers
require root permissions (e.g. ports < 1024), or if you run into compatibility
issues with non-podman clients, you can switch using the following command: 

        podman machine set --rootful


API forwarding listening on: npipe:////./pipe/docker_engine

Docker API clients default to this address. You do not need to set DOCKER_HOST.

Machine "podman-machine-default" started successfully

Starting machine "podman-machine-default"

Error: VM already running or starting

Command execution failed with exit code 125
```

### Additional context

[podman-machine-restart-errors.log](https://github.com/containers/podman-desktop/files/13560913/podman-machine-restart-errors.log)

**Comment by jeffmaury** [comment]
What is this custom image ? What if you run from CLI ?

**Comment by benoitf** [comment]
in the log I see
```
Error: VM already running or starting

Command execution failed with exit code 125
```

after is that the machine is started or not ?

**Comment by odockal** [comment]
@benoitf The machine state is still running, at least by the status shown under machine name. Though the error appears.

**Comment by rostalan** [comment]
it seems to only fail if restarted from inside the podman machine details, on the resources page restart works fine.

**Comment by lstocchi** [comment]
maybe fixed by #5168? Can you try with that? @odockal

**Comment by odockal** [comment]
@lstocchi I am sorry, right now I am trying to do finish testing and issue verification. If you can please ping me on Monday I can test it then.

**Comment by benoitf** [comment]
I think it's already fixed

**Comment by slemeur** [comment]
fixed by #5150

**Comment by odockal** [comment]
I tried once and have not seen the problem.
